src/features/barn/components/SleepingAnimalModal.tsx

```
@@ -28,6 +38,70 @@ import { useSelector } from "@xstate/react";
+
+  const content = (
+    <>
+      <img
+        src={ITEM_DETAILS[resource].image}
+        alt={resource}
+        className="w-5 mr-1"
+      />
+      <span className="text-sm whitespace-nowrap">{resource}</span>
+      <span aria-hidden="true" className="mx-1 text-sm">
+        ×
+      </span>
```

ESLint

[error] Missing JSX expression container around literal string: "×"

(react/jsx-no-literals)